jerry-core/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ message(STATUS "MEM_HEAP_SIZE_KB          " ${MEM_HEAP_SIZE_KB})
 set(INCLUDE_CORE
     "${CMAKE_CURRENT_SOURCE_DIR}"
     "${CMAKE_CURRENT_SOURCE_DIR}/api"
-    "${CMAKE_CURRENT_SOURCE_DIR}/debugger"
+    "${CMAKE_CURRENT_SOURCE_DIR}/dbg"
```

Comment:
Personally I would prefer using less namespaces, and use jerry_ prefix everywhere. However this would be too much change, so probably ecma_ namespace could be kept as well. On the longer run I would prefer removing all other namespaces at the first time when they cause any problems (e.g. if somebody complains about jmem, I would rename it to jerry_alloc_). This reduces namespace conflicts.

Regarding this change even jerry_dbg_ is better than debugger_ for me. Although longer identifiers are not a problem as long as the code is well designed. In the debugger the jerry_debugger_ never caused any problems, and the chance of symbol clash is very low. So making this shorter just for the sake of making it shorter is not exactly a good reason for me.